app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx

```diff
@@ -53,36 +71,27 @@ export default function ArboStudyPopup(record: any) {
             {/*Header section*/}
             <div className={"py-2 px-4"}>
                 <div className="text-lg font-bold">
-                   {`${record.pathogen} Estimate Study`}
+                   {`${record.pathogen} Estimate`}
                 </div>
                 <div className={"text-sm text-blue-600"}>
                     {record.url ? <a href={record.url} target="_blank" rel="noopener noreferrer"> {record.url} </a> : "NO URL"}
                 </div>
             </div>
             {/*SeroPrev section*/}
-            <div className={"flex justify-between bg-gray-200 w-full py-2 px-4"}>
-                <div className={"w-1/3 font-semibold"}>
-                    {"Seroprevalence"}: <b> {`${(record.seroprevalence * 100).toFixed(1)}%`}</b>
-                </div>
-                <div className={"w-2/3"}>
-                    {(record.sample_start_date && record.sample_end_date) && (
-                        <>
-                            {`${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`}
-                        </>)
-                    }
+            <div className={`flex justify-between bg-${record.pathogen.toLowerCase()} w-full py-2 px-4`}>
+                <div className={"font-semibold"}>
+                    {"Seroprevalence"}: <b> {`${(record.seroprevalence * 100).toFixed(1)}% (Arbovirus: ${pathogenFullString(record.pathogen)})`}</b>
                 </div>
             </div>
             {/*Content section*/}
             <div className={"py-2 px-4 max-h-[250px] overflow-auto"}>
+                {row("Date", `${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`)}
```

Comment:
```suggestion
                {row("Sampling Date Range", `${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`)}
```
If it's to your taste hahaha